### ISSUE TITLE
 Star Rating Fieldset Not Accessible

### ISSUE THREAD

**Issue by mplusb** [MachoThemes/strong-testimonials#127]
The star rating on the form is failing accessibility via a few different testing tools (https://wave.webaim.org/ and https://siteimprove.com/en-us/accessibility/) because the fieldset was missing a legend.

Error: A fieldset does not have a legend.

Currently the field name is identified with a <label> element before the fieldset opening tag.
<label>Rating</label><fieldset…><input…><label…>…..

Could the field name be placed within a <legend> element instead, within the <fieldset> element:
<fieldset…><legend>Rating</legend><input…><label…>……

**Comment by razvanaldea89** [comment]
Think the missing legend refers to the `<legend>Text</legend>`  missing inside the `fieldset` . Also, the `https://siteimprove.com/en-us/accessibility/ ` check is saying that there are empty labels ( labels with no text inside ), everything else seems to pass. 
Any POV on this @cristianraiber @giucu91 @andrei-l-cristea  ?
